packages/docs/docs/tutorial-basics/create-a-page.md

```diff
@@ -2,42 +2,308 @@
 sidebar_position: 1
 ---
 
-# Create a Page
+# Getting started
 
-Add **Markdown or React** files to `src/pages` to create a **standalone page**:
+This is a quickstart sample application that introduces some basic FHIR and Medplum concepts.
 
-- `src/pages/index.js` -> `localhost:3000/`
-- `src/pages/foo.md` -> `localhost:3000/foo`
-- `src/pages/foo/bar.js` -> `localhost:3000/foo/bar`
+[Full source code](https://github.com/codyebberson/lab-demo-js) for this sample application is available.  The core logic for the application can be found [here](https://github.com/codyebberson/lab-demo-js/blob/main/index.js)
 
-## Create your first React Page
+## Prerequisites
 
-Create a file at `src/pages/my-react-page.js`:
+You will need the following to get started:
 
-```jsx title="src/pages/my-react-page.js"
-import React from 'react';
-import Layout from '@theme/Layout';
+* Make sure you have an account and log into [Medplum](https://app.medplum.com)
+* Log in and create [Client Application](https://app.medplum.com/ClientApplication)
+* Save the Client ID and Client Secret, they will be needed later. TODO: include instructions on how to generate a client secret.
 
-export default function MyReactPage() {
-  return (
-    <Layout>
-      <h1>My React page</h1>
-      <p>This is a React page</p>
-    </Layout>
-  );
+These three requirements will need to be in place to connect
+
+```const BASE_URL = 'https://api.medplum.com/' // if you are connecting to an on premise deployment, these values may be different;
```

Comment:
Pro tip:  To get the correct syntax highlighting in GitHub, use the triple backtick followed by the syntax name.

````
```js
const BASE_URL = 'https://api.medplum.com/' // if you are connecting to an on premise deployment, these values may be different;
const MY_CLIENT_ID = 'MY_CLIENT_ID';
const MY_CLIENT_SECRET = 'MY_CLIENT_SECRET';
```
````

Details:
https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Grammars:
https://github.com/github/linguist/blob/master/vendor/README.md